src/window.rs

```diff
@@ -12,6 +12,7 @@ use crate::session_manager::{ClientState, SessionManager};
 use crate::tdlib::{Chat, ChatType};
 use crate::utils::{spawn, MESSAGE_TRUNCATED_LENGTH};
 use crate::Application;
+use gtk::gdk::Texture;
```

Comment:
This import should be moved up to the other gtk imports.

Reply:
@YuraIz Can you also just include `gtk::gdk` and then use `gdk::Texture` later please? We usually only include the last module of everything in gtk so that it's clearer from what module every definition come from.